### ISSUE TITLE
Mirror packages on Maven Central

### ISSUE THREAD

**Issue by argium** [AnormCypher/AnormCypher#74]
Would it be possible to mirror your packages on Maven central?  I have supply chain concerns regarding repo.anormcypher.org and having one less source reduces that risk.  Thanks.

**Comment by freeeve** [comment]
yes, I will work on this, though I'm afraid this package hasn't been kept up to date with the latest protocols from neo4j
